www/html/ci/application/views/landing.php

```diff
@@ -669,9 +669,9 @@ function googleTranslateElementInit() {
 <p>Combine the cuteness of mochi sticky rice balls and a classic game of speed hockey. You get Dango Puck! With the exception of the music, everything was made by me. All graphics, code, and audio clips were made by me.</p>
 
 <p><strong>Download the game for </strong>
-<a href="http://www.mediafire.com/file/ty75vbtmqx5450m/DangoPuckv1MacUniversal.zip" target="_blank" >MacOS</a>, 
-<a href="http://www.mediafire.com/file/yl1w0b261212heb/DangoPuckv1Win86.zip" target="_blank" >Windows</a>, or
-<a href="http://www.mediafire.com/file/hpelalbkzni12m2/DangoPuckv1LinuxUniversal.zip" target="_blank" >Linux Universal.</a>
+<a href="<?php echo base_url(); ?>downloads/DangoPuckv1MacUniversal.zip" download="DangoPuckv1MacUniversal" >MacOS</a>,
```

Comment:
I don't want to leave out the file extension type. Personally, I expect to see the .png or .jpeg when i see image files on my computer file system.
Just remove the renaming:
 <a href="/images/myw3schoolsimage.jpg" download>